Makefile

```diff
@@ -18,7 +18,6 @@ hooks:
 	ln -fs ../../bin/git-pre-commit.sh .git/hooks/pre-commit
 
 tidy: goimports
-	./bin/go-version-sync-check.sh
```

Comment:
This made little sense as this package is not deployable and therefore doesn't use any specific Go version.